scripts/provisioning/s3_setup

```diff
@@ -30,11 +30,14 @@ from s3confstore.cortx_s3_confstore import S3CortxConfStore
 
 # Get the machine id from confstore
 s3_confkeys_store = S3CortxConfStore(f'yaml:///opt/seagate/cortx/s3/mini-prov/s3_prov_config.yaml', 'machine_id_index')
+s3_provkeys_store = S3CortxConfStore(f'yaml:///opt/seagate/cortx/s3/conf/s3.config.tmpl.1-node', 'config_log_index')
```

Comment:
No hardcoading to s3.config.tmpl.1-node.
The config file will be passed to each of the phase